chromium_src/components/crash/core/app/crashpad.cc

```diff
@@ -11,10 +11,16 @@ namespace {
 // components\brave_vpn\browser\connection\win\brave_vpn_helper\brave_vpn_helper_crash_reporter_client.cc
 // // NOLINT Need keep it in sync
 constexpr char kBraveVPNHelperProcessType[] = "brave-vpn-helper";
+
+// Split into two places to avoid patching:
+// components\brave_vpn\browser\connection\wireguard\win\brave_vpn_wireguard_service\brave_wireguard_service_crash_reporter_client.cc
+// // NOLINT Need keep it in sync
+constexpr char kBraveWireguardProcessType[] = "brave-wireguard";
```

Comment:
sorry to changing this again, but I think this should be `brave-vpn-wireguard-service` to be in sync with the actual executable name.

Reply:
done